plugins/headlamp-plugin/package.json

```diff
@@ -176,5 +176,8 @@
     "plugins"
   ],
   "author": "Kinvolk GmbH",
-  "license": "Apache 2.0"
+  "license": "Apache 2.0",
+  "devDependencies": {
+    "@chromatic-com/storybook": "^1.4.0"
```

Comment:
is this new addon needed for the fix? or why was it added?

Reply:
I think this is a mistake, since I was trying to upgrade to version 8 where this gets added it is used for visual testing, but realized we are still on version 7 so this is not needed as of now